inst/include/child_model_simulation.hpp

```diff
@@ -76,65 +78,78 @@ void run_wlhiv_births(int t,
                 "run_wlhiv_births can only be called for model variants where run_child_model is true");
-  constexpr auto ss = StateSpace<ModelVariant>().base;
-  const auto cpars = pars.children.children;
-  const auto demog = pars.base.demography;
-
-  intermediate.children.asfr_sum = 0.0;
-  for (int a = 0; a < pars.base.options.p_fertility_age_groups; ++a) {
-    intermediate.children.asfr_sum += demog.age_specific_fertility_rate(a, t);
+  constexpr auto ss_b = StateSpace<ModelVariant>().base;
+  const auto& p_hc = pars.children.children;
+  const auto& p_dm = pars.base.demography;
+  const auto& p_op = pars.base.options;
+  const auto& c_ba = state_curr.base;
+  auto& n_ba = state_next.base;
+  auto& n_hc = state_next.children;
+  auto& i_hc = intermediate.children;
+
+  i_hc.asfr_sum = 0.0;
+  for (int a = 0; a < p_op.p_fertility_age_groups; ++a) {
+    i_hc.asfr_sum += p_dm.age_specific_fertility_rate(a, t);
   } // end a
 
-  for (int a = 0; a < pars.base.options.p_fertility_age_groups; ++a) {
-    intermediate.children.nHIVcurr = 0.0;
-    intermediate.children.nHIVlast = 0.0;
-    intermediate.children.df = 0.0;
-
-    for (int hd = 0; hd < ss.hDS; ++hd) {
-      intermediate.children.nHIVcurr += state_next.base.h_hiv_adult(hd, a, 1);
-      intermediate.children.nHIVlast += state_curr.base.h_hiv_adult(hd, a, 1);
-      for (int ht = 0; ht < ss.hTS; ++ht) {
-        intermediate.children.nHIVcurr += state_next.base.h_art_adult(ht, hd, a, 1);
-        intermediate.children.nHIVlast += state_curr.base.h_art_adult(ht, hd, a, 1);
-      } //end hTS
-    } //end hDS
-
-    intermediate.children.prev = intermediate.children.nHIVcurr / state_next.base.p_total_pop(a + 15,1);
-
-    for (int hd = 0; hd < ss.hDS; ++hd) {
-      intermediate.children.df += cpars.local_adj_factor * cpars.fert_mult_by_age(a) * cpars.fert_mult_off_art(hd) * ((state_next.base.h_hiv_adult(hd, a, 1) + state_curr.base.h_hiv_adult(hd, a, 1)) / 2);
-      //women on ART less than 6 months use the off art fertility multiplier
-      intermediate.children.df += cpars.local_adj_factor * cpars.fert_mult_by_age(a) * cpars.fert_mult_off_art(hd) * ((state_next.base.h_art_adult(0, hd, a, 1) + state_curr.base.h_art_adult(0, hd, a, 1)) / 2);
-      for (int ht = 1; ht < ss.hTS; ++ht) {
-        intermediate.children.df += cpars.local_adj_factor * cpars.fert_mult_on_art(a) * ((state_next.base.h_art_adult(ht, hd, a, 1) + state_curr.base.h_art_adult(ht, hd, a, 1)) / 2);
-      } //end hTS
+  for (int a = 0; a < p_op.p_fertility_age_groups; ++a) {
+    i_hc.nHIVcurr = 0.0;
+    i_hc.nHIVlast = 0.0;
+    i_hc.df = 0.0;
+
+    for (int hd = 0; hd < ss_b.hDS; ++hd) {
+      i_hc.nHIVcurr += n_ba.h_hiv_adult(hd, a, 1);
+      i_hc.nHIVlast += c_ba.h_hiv_adult(hd, a, 1);
+      for (int ht = 0; ht < ss_b.hTS; ++ht) {
+        i_hc.nHIVcurr += n_ba.h_art_adult(ht, hd, a, 1);
+        i_hc.nHIVlast += c_ba.h_art_adult(ht, hd, a, 1);
+      } // end hTS
```

Comment:
minor point but comment spacing consistent so `// end hTS` instead of `//end hTS`